opentelemetry-etw-metrics/src/lib.rs

```diff
@@ -1,4 +1,4 @@
-mod etw;
+pub mod etw;
```

Comment:
we won't want to make this public. lets revert if this was done for some testing.

Reply:
I made it public so that I could access `etw::write` in a benchmark. Is there a better way to achieve that?

Reply:
no. and I don't that is something we need to test either. cost of writing to etw is indirectly covered by the other benchmark.

Reply:
Great, remove the `etw` benchmark and restored the `etw` module's privacy